pacstrap: add page (#5247)
tldr-pages/tldr: pages/linux/pacstrap.md

# pacstrap

> Install packages using Pacman to the specified new root directory.
> More information: <https://man.archlinux.org/man/pacstrap.8>.

- Install the `base` package, Linux kernel and firmware for common hardware:

`pacstrap {{path/to/new/root}} {{base}} {{linux}} {{linux-firmware}}`

> - Install packages without copy the host’s mirrorlist to the target:

Reviewer: ```suggestion
- Install packages without copy the host's mirrorlist to the target:
```

`pacstrap -M {{path/to/new/root}} {{packages}}`

- Use an alternate configuration file for Pacman:

`pacstrap -C {{path/to/pacman.conf}} {{path/to/new/root}} {{packages}}`

> - Install packages using the package cache on the host, rather than the target:

Reviewer: At first I was confused by this statement.  I think this is clearer:
```suggestion
- Install packages using the package cache on the host instead of on the target:
```

`pacstrap -c {{path/to/new/root}} {{packages}}`

- Install packages without copy the host's pacman keyring to the target:

`pacstrap -G {{path/to/new/root}} {{packages}}`

> - Install packages in interactive mode (prompts user for confirmation):

Reviewer: The reader _is_ the user :)
```suggestion
- Install packages in interactive mode (prompting for confirmation):
```

Author: I used `prompts` from https://github.com/tldr-pages/tldr/blob/master/pages/linux/cp.md page

Reviewer: Hmm...well maybe we change it there and everywhere, what do you think?  Because though intuitive to write, from a reader's/user's perspective, it may look a little awkward.  @sbrl

Reviewer 2: I'd just say `prompts for confirmation` to be honest. 🤷🏻

Reviewer: I like that too.
```suggestion
- Install packages in interactive mode (prompts for confirmation):
```

`pacstrap -i {{path/to/new/root}} {{packages}}`

- Install packages using package files:

`pacstrap -U {{path/to/new/root}} {{path/to/package1}} {{path/to/package2}}`

> - Install the `base` package, Linux LTS kernel and `base-devel` build tools:

Reviewer: I think you should move this example up, right below the first one, because it's very similar.
